Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -107,6 +109,30 @@ private function createReferenceRelationTable(): Table
             ->setPrimaryKey(['name', 'position', 'nodeanchorpoint']);
     }
 
+    private function createWorkspaceTable(): Table
```

Comment:
💡 i compared the schemas again against the current ones and can confirm besides a few renames of columns its the same. Thus the migration in setup can definitely work.